.travis.yml

```diff
@@ -1,6 +1,8 @@
-sudo: required
-dist: bionic
-language: python
```

Comment:
`language: python` is necessary, otherwise the below `python: 3.5` is ignored (and it then uses deprecated python2).